### ISSUE TITLE
Wrong packages on Centos 6.9

### ISSUE THREAD

**Issue by alexmocanu** [nodesource/distributions#450]
When I try to install nodejs 7 on Centos 6.9 for some reason it installs the packages for nodejs 6

I removed any existing nodejs packages:
- yum remove nodesource-release* nodejs

I cleaned the yum  cache:
- yum clean all

I removed any nodesource files in /etc/yum.repos.d
- rm -fv /etc/yum.repos.d/nodesource*

Then i ran this line:
- curl --insecure --silent --location https://rpm.nodesource.com/setup_7.x | bash -

And finally I tried to install nodejs using this command:
- yum install nodejs

This is when I got a lot of errors while the script tried different mirrors:
PYCURL ERROR 22 - "The requested URL returned error: 404 Not Found"

And finally it installed  nodejs.x86_64 2:6.10.2-2nodesource.el6 instead of any 7.x package...

**Comment by skolesnyk** [comment]
Same here. Doesn't want to install node 7.x , only installs nodejs 6.10.3

**Comment by d3x0r** [comment]
#340  has a lot of steps about clearing cache, removing excessive repos etc.
I'm trying to install 8, not 7.

finally got it to work with this series of commands....
I was going to comment that I wasn't able to get it to work; and that it was still getting 0.10.something but I guess I wasn't doing all the steps together first.
in the other thread it was 'yum remove nodesource-release' which I guess wasn't getting everything.

again, to be clear; this worked.
```
yum remove nodesourc*
yum clean all 
rm -fr /var/cache/yum/*
curl -sL https://rpm.nodesource.com/setup_8.x | bash -
yum install nodejs
```

**Comment by okonon** [comment]
@d3x0r commands did not work for me.

**Comment by d3x0r** [comment]
*shrug* I was able to use that on 4 boxes in a row (well 3 after doing the original commands that didn't work, because the yum repo changed from nodesource-release to just nodesource) I made it as a script and copied to each box and ran the script and viola upgraded node.   they were fairly clean boxes; though they did already have gcc and related build tools installed.  
Look in /etc/yum.conf and /etc/yum.repos.d/  
the only other place yum has stuff is /var/cache/yum which the script clears.
